src/Resources/app/storefront/src/configuration/adyen.js

```diff
@@ -146,6 +146,7 @@ export default {
         'afterpay_default': 'handler_adyen_afterpaydefaultpaymentmethodhandler',
         'trustly': 'handler_adyen_trustlypaymentmethodhandler',
         'paysafecard': 'handler_adyen_paysafecardpaymentmethodhandler',
+        'giftcard': 'handler_adyen_giftcardpaymentmethodhandler',
         'givex': 'handler_adyen_givexgiftcardpaymentmethodhandler',
```

Comment:
do we still need all the other giftcards to be in this list? 🤔

Reply:
That we are still not sure, so kept it for now. Once we are done with the actual /payments call, then we can decide over it